packages/common/src/types/schema.ts

```diff
@@ -401,34 +385,36 @@ export type InvoiceLinesResponse = ConnectorError | InvoiceLineConnector;
 
 export type InvoiceNode = {
   __typename?: 'InvoiceNode';
+  allocatedDatetime?: Maybe<Scalars['DateTime']>;
+  color: Scalars['String'];
   comment?: Maybe<Scalars['String']>;
   confirmedDatetime?: Maybe<Scalars['DateTime']>;
+  deliveredDatetime?: Maybe<Scalars['DateTime']>;
+  draftDatetime?: Maybe<Scalars['DateTime']>;
   entryDatetime: Scalars['DateTime'];
   finalisedDatetime?: Maybe<Scalars['DateTime']>;
+  hold: Scalars['Boolean'];
   id: Scalars['String'];
   invoiceNumber: Scalars['Int'];
   lines: InvoiceLinesResponse;
   otherPartyId: Scalars['String'];
   otherPartyName: Scalars['String'];
+  pickedDatetime?: Maybe<Scalars['DateTime']>;
   pricing: InvoicePriceResponse;
+  shippedDatetime?: Maybe<Scalars['DateTime']>;
   status: InvoiceNodeStatus;
   theirReference?: Maybe<Scalars['String']>;
   type: InvoiceNodeType;
 };
 
 export enum InvoiceNodeStatus {
-  /**
-   * For customer invoices: When an invoice is CONFIRMED available_number_of_packs and
-   * total_number_of_packs get updated when items are added to the invoice.
-   */
+  Allocated = 'ALLOCATED',
   Confirmed = 'CONFIRMED',
-  /**
-   * For customer invoices: In DRAFT mode only the available_number_of_packs in a stock line gets
-   * updated when items are added to the invoice.
-   */
+  Delivered = 'DELIVERED',
   Draft = 'DRAFT',
-  /** A FINALISED invoice can't be edited nor deleted. */
```

Comment:
I guess it initially generated with the comments and then decided it didn't want to anymore..